pyprep/utils.py

```diff
@@ -121,6 +144,37 @@ def _get_random_subset(x, size, rand_state):
     return sample
 
 
+def _correlate_arrays(a, b, axis=1):
```

Comment:
can we call this _mat_correlate_arrays? Or something the like? So that we better see that this comes from Matlab PREP?

Reply:
So far I've only used the `_mat` prefix for functions that mimic the behaviour of built-in MATLAB functions. I can add a `_matprep` prefix to differentiate, but I was thinking for my follow-up PR that I'd make `_correlate_arrays` like `_predict_median_signals` and add a `matlab_strict` flag to switch between PyPREP (`np.corrcoef`) and MatPREP behaviour.

Reply:
>  but I was thinking for my follow-up PR that I'd make _correlate_arrays like _predict_median_signals and add a matlab_strict flag to switch between PyPREP (np.corrcoef) and MatPREP behaviour.

okay, as long as you don't forget about that, that sound better :+1:

Reply:
I was thinking that maybe it was better using the whole "matlab" prefix instead of the "mat" one. I understand is more verbose but one could confuse "mat" as being associated with a matrix.

In any case, this is just a minor suggestion, is not that relevant.